app/services/session.js

```diff
@@ -59,7 +59,9 @@ export default Service.extend({
       .then(userPayload => {
         if (userPayload.errors) {
           const errors = this.processLoginErrors(userPayload.errors);
-          return { errors };
+          return {
```

Comment:
this was an auto format, i had a problem with the session when the back end resets the local storage token becomes invalid, i'll open a PR to fix that later.